ego/templates/deployment.yaml

```diff
@@ -39,7 +39,7 @@ spec:
             - name: SPRING_SECURITY_OAUTH2_CLIENT_PROVIDER_GOOGLE_AUTHORIZATIONURI
               value: {{ .Values.appConfig.googleAuthorizationUri }}
             - name: SPRING_SECURITY_OAUTH2_CLIENT_REGISTRATION_GOOGLE_REDIRECTURI
-              value: "https://{{ .Values.appConfig.host }}/oauth/code/google"
+              value: "https://{{ .Values.appConfig.host }}/api/oauth/code/google"
```

Comment:
instead of hardcoding `/api`, can we use a reference to the variable we set, so this is always matching up?

Reply:
Done!